db/db_test.go

```diff
@@ -334,6 +335,75 @@ var _ = Describe("Lightnode db", func() {
 					Expect(quick.Check(test, &quick.Config{MaxCount: 10})).NotTo(HaveOccurred())
 				})
 			})
+
+			Context("when querying txs with a given tag", func() {
```

Comment:
It would be prudent to also have some negative tests where

1. malformed or otherwise bizarre tags are used, and
2. we query for a tag that has no transaction associated with it.